.github/workflows/provider.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
           cp ./lakefs_provider/example_dags/lakefs-dag.py astro/dags/
 
       - name: Insert Dockerfile pip install
-        run: printf "\nRUN pip install --user dist/airflow_provider_lakefs-0.41.0.1-py3-none-any.whl" >> astro/Dockerfile
+        run: printf "\nRUN pip install --user dist/airflow_provider_lakefs-0.41.0.3-py3-none-any.whl" >> astro/Dockerfile
```

Comment:
why don't we pass the version as build arg? we need a final static Dockerfile, or just push the result?

Reply:
Sorry for pushing and missing this comment.
It's determined in the `setup.py` file, where version isn't injected too.
I'll change it to inject to both in a new PR.

Reply:
Wait, I can't really add env var to workflows. Any other dynamic way for doing so??